spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -215,8 +215,6 @@ describe('document-capture/components/document-capture', () => {
 
     submitButton = getByText('forms.buttons.submit.default');
     await userEvent.click(submitButton);
-    const interstitialHeading = getByText('doc_auth.headings.interstitial');
```

Comment:
There seems to be some timing changes where the interstitial is already removed by the time we get here. I don't think it's critical for us to ensure that the interstitial was visible, more that the expected alert message eventually shows up.